packages/wallet-core/src/web-wallet-channel/requests.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as v from 'valibot';
+
+export type JsonData = string | number | boolean | null | { [key: string]: JsonData } | JsonData[];
+
+const JsonSchema: v.GenericSchema<JsonData> = v.lazy(() =>
+	v.union([
+		v.string(),
+		v.number(),
+		v.boolean(),
+		v.null(),
+		v.record(v.string(), JsonSchema),
+		v.array(JsonSchema),
+	]),
+);
+
+export const RequestData = v.variant('type', [
+	v.object({
+		type: v.literal('connect'),
+		chain: v.string('`chain` is required'),
```

Comment:
added chain to connect as well and until we have it in the connect request we can default it with the network of the wallet-standard ie [here](https://github.com/MystenLabs/ts-sdks/pull/83/files#diff-c9f674dd4746d5e05a9340d7ac3a2f50287e2734ed0672b086ea19562b4b88ffR225)

Reply:
Connect should be chain-agnostic?

Reply:
I added this mainly for zklogin that we usually create a jwt with ephemeral private key based on the selected network. But I think you are right chain is not required for connect. I will remove it